### DISCUSSION TITLE
How are the source and destination network prefixes computed?

### DISCUSSION THREAD

**Discussion by ghost** [akvorado/akvorado#1260] [Q&A]
Hi, 

I see a lot of flows with a /40, /63 or even ::/0 in the source or destination network prefix field and it is a rare number, so I was wondering how Akvorado calculates this parameter, since in the routing router there is a different netmask declared.

I'm digging into a flow that Akvorado assigned ::/0 as the destination network prefix to see if I find anything strange. In this case, in the routing table we have 2 entries: the first one is the destination network and the second one is ::/0 because we have several VRFs and the network is not propagated to all of them. I was wondering if perhaps Akvorado gets confused by having multiple entries in the routing table and assigns ::/0 instead of the other. Any ideas?

Thanks for your time!

**Comment by vincentbernat** [comment]
If you didn't set a BMP receiver, these informations are part of the captured flows (they contain a source and destination prefix length).
